setup.py

```diff
@@ -230,7 +230,8 @@ def setup_package():
         FULLVERSION, GIT_REVISION = get_version_info()
         metadata['version'] = FULLVERSION
     else:
-        if len(sys.argv) >= 2 and sys.argv[1] == 'bdist_wheel':
+        if (len(sys.argv) >= 2 and sys.argv[1] == 'bdist_wheel') or (
+                    'develop' in sys.argv):
```

Comment:
The ``if`` clause above is only for a collection of commands that use ``setuptools`` but do *not* build scipy.